openmc/tallies.py

```diff
@@ -1653,7 +1659,8 @@ def _align_tally_data(self, other):
             if self_repeat_factor == 1:
```

Comment:
Do you mean lines 1661-1662? I'm actually not sure, it may be a bug that's not yet been exposed. Of course this section of code is pretty fundamental to tally arithmetic, so I would be hesitant to say that there is not a reason for repeating the `other` array here. I do recall observing some odd behaviour when tiling and then repeating an array, vs. repeating an tiling an array, and the scheme here may be a result of that.